llvm/lib/Target/WebAssembly/WebAssemblyAsmPrinter.cpp

```diff
@@ -249,13 +249,6 @@ MCSymbol *WebAssemblyAsmPrinter::getOrCreateWasmSymbol(StringRef Name) {
   SmallVector<wasm::ValType, 4> Params;
   if (Name == "__cpp_exception" || Name == "__c_longjmp") {
```

Comment:
I guess this means we are still special casing the symbol types for these symbols based on their name, right? How does this relate to the `.tagtype` asm directive? I guess that doesn't set the symbol type, it just sets the signature of the tag itself?

Reply:
Not sure if meant this, but our asm parser sets the tag type when parsing `.tagtype`:
https://github.com/llvm/llvm-project/blob/fdb1f486387d46bd046d7827fcd19fa44118bffe/llvm/lib/Target/WebAssembly/AsmParser/WebAssemblyAsmParser.cpp#L1096